prow/plugins/blockade/blockade.go

```diff
@@ -74,9 +74,30 @@ func helpProvider(config *plugins.Configuration, enabledRepos []config.OrgRepo)
 		}
 		blockConfig[repo.String()] = buf.String()
 	}
+	yamlSnippet, err := plugins.CommentMap.GenYaml(&plugins.Configuration{
+		Blockades: []plugins.Blockade{
+			{
+				Repos: []string{
+					"ORGANIZATION",
+					"ORGANIZATION/REPOSITORY",
+				},
+				BlockRegexps: []string{
+					"^examples/",
+				},
+				ExceptionRegexps: []string{
+					"^excepts/",
```

Comment:
> ExceptionRegexps are regular expressions matching the file paths that are exceptions to the BlockRegexps.

Thus ExceptionRegexps only make sense if they match a subset of what the BlockRegexps match. So this example is misleading.